.changeset/tasty-carpets-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-react': minor
+---
+
+**BREAKING**: Removed `useEntityKinds` hook.
```

Comment:
should probably mention "please use `catalogApi.getEntityFacets({ facets: ['kind'] })` instead" or so

Reply:
good point 👍